buildscripts/build.cmd

```diff
@@ -40,7 +41,9 @@ dotnet restore ./src/Castle.Windsor.Tests/Castle.Windsor.Tests.csproj
 GOTO build
 
 :build
+dotnet build ./tools/Explicit.NuGet.Versions/Explicit.NuGet.Versions.sln
 dotnet build Castle.Windsor.sln -c %Configuration%
+.\tools\Explicit.NuGet.Versions\build\nev.exe ".\build" "castle."
```

Comment:
Won't this also incorrectly update Castle.Core?

Reply:
Don't we want explicit versioning for all of it? Not a problem just need to add more commands to target specific package id's in the solution.

Reply:
We've only discussed explicit versioning within a project, however I **don't** think Windsor should require a specific version of Castle Core assemblies.

Reply:
All fixed

![castle-windsor](https://user-images.githubusercontent.com/607402/30606200-ccd64c62-9d68-11e7-8807-bf3ccab4951d.png)

![castle-loggingfacility](https://user-images.githubusercontent.com/607402/30606206-d055f3e2-9d68-11e7-9465-0e79c7cb85c6.png)